fhir_zulip_nlp/fhir_zulip_nlp.py

```diff
@@ -35,6 +30,7 @@
     # 'outdir': PROJECT_DIR,
     # 'report1_filename': 'fhir-zulip-nlp - frequencies and date ranges.csv',
     # 'report2_filename': 'fhir-zulip-nlp - age and date analyses.csv',
+    # TODO: these types of files should have a '.' in front of them, so will change.
```

Comment:
@rohaher FYI; I mentioned this earlier but will probably change back to `.zuliprc` at some point